website/docs/components/stepper/nav/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,63 @@
+## Usage
+
+### When to use
+
+- In complex flows that are broken up into multiple steps to more easily segment content.
+- To communicate progress in a mult-step flow; what has been completed, and what has yet to be completed.
```

Comment:
removing repetition

```suggestion
- To communicate progress in a mult-step flow.
```

Or something like:
"To communicate what has and has not been completed in a mult-step flow."